server/spec/utils/index.js

```diff
@@ -302,6 +318,16 @@ exports.cleanDatabase = async function() {
   logger.debug(`Cleaned database in ${duration}s`);
 };
 
+/**
+ * Deletes all test emails that were sent so far.
+ *
+ * This should be used before every test by calling `clean` (typically in a
+ * `beforeEach` hook).
+ */
+exports.cleanEmails = function() {
+  testMails.length = 0;
```

Comment:
It would not work.

When `testMails` is [required at the top](https://github.com/MediaComem/biopocket-backend/blob/a864b60e782a654489e12df53a71864ff8d8afef/server/spec/utils/index.js#L14), a new `testMails` variable is created in this module that points to the same array as the one  [exported from the mailer](https://github.com/MediaComem/biopocket-backend/blob/a864b60e782a654489e12df53a71864ff8d8afef/server/utils/mailer.js#L13). However, although the two variables point to the same array in memory at that time, they are **two distinct variables** in separate modules. Re-assigning the value of one will not update the value of the other. That is why we have to empty the existing array, not create a new one.